src/core/ext/client_config/resolver_registry.c

```diff
@@ -108,7 +114,7 @@ static grpc_resolver_factory *resolve_factory(const char *target,
   *uri = grpc_uri_parse(target, 1);
   factory = lookup_factory_by_uri(*uri);
   if (factory == NULL) {
-    if (g_default_resolver_prefix != NULL) {
+    if (g_default_resolver_prefix[0] != '\0') {
```

Comment:
Now that there is a built-in default prefix, is there any case where we would expect the value to be empty?